
class PyDataCredentialsError(ValueError):
    """
    Raised when invalid credentials are provided, or tokens have expired.
    """
